README.md

```diff
@@ -57,103 +57,136 @@ class Apollo apolloBox
+1. Visit the [public Apollo Explorer instance](https://studio.apollographql.com/public/spotify-ev3of9/variant/prod/home) to interact with the graph (No GraphOS account required)
 
 - [Query the graph](https://studio.apollographql.com/public/spotify-ev3of9/variant/prod/explorer) _(**Spotify account required**) - OAuth workflow with be initiated from Apollo Explorer to login to our Spotify account to run any operation_
   - Try having your Spotify app playing on your phone or desktop and then run [this mutation](https://studio.apollographql.com/public/spotify-ev3of9/variant/prod/explorer?explorerURLState=N4IgJg9gxgrgtgUwHYBcQC4RxighigSwiQAIAFXGAZwTIBtcBPAI1ygGsBZHfI04ADqkSIgA6Ua9Jqw4lBwkWIYs27AMp4UCOUMV6SBKlMYEkAc136AvpZE3hVkFaA)
 - [View the graph's schema](https://studio.apollographql.com/public/spotify-ev3of9/variant/prod/schema/reference)
 
 ### I want to re-create this demo in my GraphOS account
 
-3. Create a [personal API key](https://www.apollographql.com/docs/graphos/org/account/#personal-api-keys)
-4. Run the clone script with the API key
+1. Clone this repo.
+2. Create a [personal API key](https://www.apollographql.com/docs/graphos/org/account/#personal-api-keys).
+3. Run the script below using your personal API key.
 
-```sh
-APOLLO_KEY={YOUR_API_KEY} npm run graphos-demo
-```
+   ```sh
+   AUTH={YOUR_API_KEY} npm run graphos-demo
+   ```
+
+   If you have multiple organizations, the script will prompt you to select which organization the graph should be in.
 
 ### I want to run the client app locally
 
-3. Install dependencies
+1. Clone this repo.
+2. From the root of the folder, install the dependencies.
 
-```sh
-npm install
-```
+   ```sh
+   npm install
+   ```
 
-4. Start the client app
+3. From the root of the folder, start the client app.
 
-```sh
-npm start
-```
+   ```sh
+   npm start
+   ```
+
+4. Visit the website at http://localhost:3000.
 
-5. Visit the website at http://localhost:3000
+By default, the client app is pointing to the _locally-running_ backend URL. Follow the section below for "I want to run the backend locally" (you can choose to use Docker, or not).
 
-**\*Note**: We're currently working on subscriptions support with `rover dev` so the app is pointing at the deployed production url when running locally. You can change the URL the client application is pointing at by editing the `.env.development` file with `VITE_SERVER_HOST` set to the desired URL (most likely locally in the next steps)
+Local subscriptions using `rover dev` is currently a work-in-progress, so features that use subscriptions are not functional (such as listening to playback state). You can change the URL the client application is pointing at by editing the `.env.development` file with `VITE_SERVER_HOST` and setting it to the production URL (uncomment the second line).
```

Comment:
The instructions are actually downloading the router and running it. Do local subscriptions work that way? Should we remove this `rover dev` note?